src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -179,13 +179,14 @@ private bool CrossgenAssembly(string publishFolder, PackageEntry package, Librar
             return exitCode == 0;
         }
 
-        private DirectoryInfo CreateSubDirectory(string cacheBasePath, PackageEntry package, LibraryAsset asset)
+        private DirectoryInfo CreateSubDirectoryInLowerCase(string cacheBasePath, PackageEntry package, LibraryAsset asset)
         {
             var subDirectoryPath = Path.Combine(
                 cacheBasePath,
                 package.Library.Identity.Name,
                 package.Library.Identity.Version.ToNormalizedString(),
-                Path.GetDirectoryName(asset.RelativePath));
```

Comment:
No, but CLI `rel/1.0.0` branch is depending on `3.6.0-beta.1.msbuild.9`, which has the proper functionality:
https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.Cli.Utils/project.json